e2e/cypress/integration/enterprise/guest_accounts/guest_invitation_ui_spec.js

```diff
@@ -315,4 +319,45 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         // * Verify the content and message in next screen
         verifyInvitationError(email, 'Error: Email invitations are disabled.');
     });
+
+    it('MM-18047 Verify Guest User whitelisted domains', () => {
+        // #Configure a whitelisted domain
+        changeGuestFeatureSettings(true, true, 'example.com');
+
+        // # Login as sysadmin and visit to newly created team
+        cy.apiLogin('sysadmin');
+        cy.visit(`/${testTeam.name}`);
+
+        // # Invite a Guest by email
+        const email = `temp-${getRandomInt(9999).toString()}@mattermost.com`;
+        invitePeople(email, 1, email);
```

Comment:
Noticed that this function is fed the same `email`. Are there scenarios where you would use different emails for the arguments?

Reply:
Yes. The second argument is what gets displayed in the dropdown. If a user exists, then the user would be displayed, else it would display the email.

Reply:
Bah, nevermind. I must have had a section collapsed when I last searched, I see examples where you are feeding in different emails/names.